uber/utils.py

```diff
@@ -336,26 +340,46 @@ def to_dict(self):
         }
 
     @property
-    def models(self):
-        return self._models_cached
+    def is_empty(self):
+        return not self._targets
 
-    @property
+    @cached_property
```

Comment:
Does this mean that the total_cost for a specific Charge() object will never be recalculated for that object? We definitely expect this property to be able to change elsewhere in the code: https://github.com/magfest/ubersystem/blob/master/uber/site_sections/preregistration.py#L307

While it isn't a problem for most cases, I just want to make sure that if someone starts a transaction on DAY_BEFORE_PRICE_BUMP and leaves their screen in place till DAY_AFTER_PRICE_BUMP we'll still catch that. I don't have a good model of how cached properties work in my head, so sorry in advance if this is a silly concern!